net/lighttpd-https-cert/files/ssl-enable.conf

```diff
@@ -4,14 +4,22 @@
 # This package is not meant to be edited, it is part of package lighttpd-https-cert
 # If you need different https setup, uninstall this package and create your own configuration
 
+# As a special case in lighttpd 1.4.46 and later, if ssl.engine = "enable" in
+# the global scope and along with ssl.pemfile, then $SERVER["socket"] blocks
+# can inherit the exact global config by setting ssl.engine = "enable" and no
+# other ssl.* directives in the $SERVER["socket"] block.
+ssl.engine = "enable"
+ssl.pemfile = "/etc/lighttpd-self-signed.pem"
+ssl.use-sslv2 = "disable"
+ssl.use-sslv3 = "disable"
```

Comment:
From [lighttpd documentation](https://redmine.lighttpd.net/projects/lighttpd/wiki/Docs_SSL#Configuration):
> ssl.use-sslv2 | enable/disable use of SSL version 2 (lighttpd < 1.4.21 only, newer version don't support SSLv2)
> ssl.use-sslv3 | enable/disable use of SSL version 3 (lighttpd >= 1.4.29 only)

Since we have Lighttpd version 1.4.50 at the moment I think this is not needed.

AFAIK SSL2 is not compiled in openssl library so it should not exists on our routers at all.

Reply:
As discussed in [the forums](https://forum.turris.cz/t/ssl-tls-interface-default-configuration/10888/10) I changed this to a more generic (but stricter) configuration with the `ssl.openssl.ssl-conf-cmd`.